src/utils/renderHelpers.ts

```diff
@@ -5,6 +5,8 @@ import { DynamicImportMechanism } from '../Chunk';
 export interface RenderOptions {
 	legacy: boolean;
 	freeze: boolean;
+	toStringTag: boolean;
```

Comment:
Maybe we could use a name that tells us a little more in what situations this causes a change? E.g. something like `namespaceToString` as it is adding a `toString` to namespace variables?